README.md

```
@@ -51,13 +51,61 @@ A: This a real-time, user-friendly `addition` to the classical methods, not a co
-kubectl apply -f example/redis-client/client.yaml  
+**5. Run the attack suite with bobctl** — RESP + exec kill-chain against redis-demo.
+`redis.yaml` ships the **redis-vulnerable** image, so the Lua-escape / RediShell
+exploits actually execute; the bound SBoB then catches them (R0001 spawned
+`sh`/`perl`, R0002 payload file reads) while the benign functional suite stays at
+zero false positives:
+```bash
+bobctl attack --attack-suite example/redis-attacks.yaml -n redis-demo --service redis --service-port 6379 --format table
+```
+
+**Auto-tune it** — learn benign (functional suite) + run attacks + minimise with wildcards, scored:
+```bash
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)